### ISSUE TITLE
investigate whether web3.near.retrieveNear is adjusting near balance properly

### ISSUE THREAD

**Issue by emaG3m** [nearprotocol/near-web3-provider#42]
When calling retrieve near, the ethereum balance is clearly subtracted properly. However the near account balance doesn't seem to change when investigating through: 

```javascript
let nearBalance = await web._provider.account.getAccountBalance()
nearBalance.total           // no difference
nearBalance.available    // no difference
```

**Comment by barbaraliau** [comment]
@emaG3m were you able to resolve this?

**Comment by emaG3m** [comment]
> @emaG3m were you able to resolve this?

no. was going to poke around with `near-evm` tests (https://github.com/near/near-evm/blob/master/tests/cryptozombies_rpc.rs) to compare, but those tests are out of whack now, I'm guessing due to nearcore updates. I have a hunch this has to do with the `near-evm` side of things though rather than the web3 provider,
